README.md

```diff
@@ -22,19 +22,31 @@
 
 </div>
 
-Features
---------
+## Features
 
-**collective.casestudy** provides a Case Study content type to be used in Plone sites.
+**collective.casestudy** provides two new content types to be used in Plone sites:
```

Comment:
All instances of `collective.casestudy` should be changed from bold format to an inline literal. I made just this one suggestion to not repeat myself.

```suggestion
`collective.casestudy` provides two new content types to be used in Plone sites, Case Study and Provider.
```